docs/site/Repositories.md

```diff
@@ -41,11 +51,14 @@ app.repository(CategoryRepository);
 
 ## Configure datasources
 
-You can define a DataSource using legacy Juggler in your LoopBack 4 app as follows:
+You can define a DataSource using legacy Juggler in your LoopBack 4 app as
+follows:
 
-```js
+```ts
+// src/datsources/db.datasource.ts
 import {juggler, DataSourceConstructor} from '@loopback/repository';
 
+// this is just an example, 'test' database doesn't actually exist
```

Comment:
not sure if there's a need to mention that the db is an example :)

Reply:
I personally didn't know, tried it out, and found out it didn't exist. Maybe the notice should be worded differently?